data/repo.xml

```diff
@@ -24,9 +24,9 @@
                 </listitem>
             </itemizedlist>
             <para>This split made it difficult to redistribute applications. For larger setups,
-            maintenance easily became tedious. To solve those problems, eXist-db 2.0 introduces the
+            maintenance easily became tedious. To solve those problems, eXist-db 2.0 introduced the
```

Comment:
To solve those problems, eXist-db has the  concept of self-contained, modular applications which can be deployed into any database....